Makefile

```diff
@@ -15,7 +15,7 @@ test: .install flake8
 	py.test -v tests/
 
 cov cover coverage: .install flake8
-	py.test -v --cov=datarobot_batch_scoring \
+	py.test -v -x --cov=datarobot_batch_scoring \
```

Comment:
it'll fail after first failed test. I don't think we want this at travis builds (annoying if you have two failed tests and it'll be shown in two builds, not one)